support/cas-server-support-thymeleaf/src/main/resources/templates/inwebo/casInweboBrowserAuthentView.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
```

Comment:
HTML pages should be renamed to either fully refer to "Authntication", or shortened for "Authn". "Authent" seems quite odd to me.

Reply:
Yes, indeed, it sounds odd. Will rename.